src/app/application.cpp

```diff
@@ -662,6 +662,24 @@ void Application::sendNotificationEmail(const BitTorrent::Torrent *torrent)
                      content);
 }
 
+void Application::sendTestEmail()
```

Comment:
```suggestion
void Application::sendTestEmail() const
```

Reply:
Thanks for the review. Will get these changes pushed.